.babelrc

```diff
@@ -1,12 +1,28 @@
 {
 	"presets": ["env"],
-	"plugins": ["transform-runtime"],
+	"plugins": [
+		"transform-runtime",
+		["module-resolver", {
+				"root": ["./src"]
+			}
+		]
+	],
 	"env": {
 		"test": {
 			"plugins": ["istanbul", "rewire"]
 		},
 		"browsertest": {
 			"plugins": ["rewire"]
+		},
+		"buildcheck": {
+			"plugins": [
+				["module-resolver", {
+					"root": ["./dist-node"]
+					}
```

Comment:
Formatting is odd here. Maybe we should add this file to our Prettier settings (or rename it to `.json` if that's possible)?